android/java/org/chromium/chrome/browser/crypto_wallet/adapters/RecoveryPhraseAdapter.java

```diff
@@ -69,45 +71,46 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
         }
     }
 
-    public void setSelectedRecoveryPhrase(boolean isSelectedRecoveryPhrase) {
-        this.isSelectedRecoveryPhrase = isSelectedRecoveryPhrase;
+    public void setSelectedRecoveryPhrase(final boolean selectedRecoveryPhrase) {
+        this.mSelectedRecoveryPhrase = selectedRecoveryPhrase;
```

Comment:
I don't think we need `this` anymore if we changed members to start with `m`

Reply:
👍